src/lib.rs

```diff
@@ -421,6 +424,17 @@ impl TxBuilder {
         })
     }
 
+    fn set_recipients(&self, recipients: Vec<AddressAmount>) -> Arc<Self> {
```

Comment:
The addition of the `AddressAmount` type here is a good thing (I ended up doing the same thing manually on the Kotlin side), and I'm wondering if we should also use that type for the `add_recipient()` method above.

Reply:
We are forced to use `AddressAmount` to work in a vector for `add_recipients`.. but for `add_recipient` I think it's best to leave it as it is since it's trivial to still use a `AddressAmount` struct by using it's fields.